packages/mobile/components/StrongholdUnlock.svelte

```diff
@@ -0,0 +1,58 @@
+<script lang="typescript">
+    import { Button, PasswordInput, Text, HTMLButtonType } from 'shared/components'
```

Comment:
Not sure if this component should be rather inside the drawers/account-switcher/views folder.